main.py

```diff
@@ -2,6 +2,65 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actor_data = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    Genre.objects.bulk_create(
+        [
+            Genre(name=genre)
+            for genre in genres
+        ]
+    )
+    Actor.objects.bulk_create(
+        [
+            Actor(
+                first_name=actor[0],
+                last_name=actor[1],
+            )
+            for actor in actor_data
```

Comment:
you can do:
for first_name, last_name in actor_data

Reply:
Got it. Thanks!